src/nrnpython/nb_defs.h

```diff
@@ -0,0 +1,7 @@
+#pragma once
+
+#if defined(_WIN32)
+#define NB_EXPORT __declspec(dllexport)
+#else
+#define NB_EXPORT __attribute__((visibility("default")))
```

Comment:
Why are we defining a macro that nanobind is already using?

Reply:
because inithoc.cpp is compiled in a weird way, that is not including nanobind (and I don't really understand why).

Reply:
Discussed offline. Summary: not a fight worth picking right now.